stable/app/templates/deployment.yaml

```diff
@@ -22,6 +22,7 @@ spec:
         {{- end }}
       labels:
         {{- include "app.selectorLabels" . | nindent 8 }}
+        app.pixxel.co.in/governed-by: "deployment"
```

Comment:
Should ideally go as additional labels as input params instead of hard-coding anything business specific

Reply:
Issue with it being input labels, is same input label will be applied to both deployment container and migration-job, resulting in the pdb error, I mentioned before.